docs/conf.py

```diff
@@ -4,6 +4,8 @@
 # list see the documentation:
 # https://www.sphinx-doc.org/en/master/usage/configuration.html
 
+# type: ignore
```

Comment:
I just removed `pytest.ini` without this comment and it didn't seem to complain. Perhaps it's not necessary?

Reply:
Yup, some info here: https://github.com/python/mypy/issues/964

Reply:
When I removed `pytest.ini` I got failures from `pytest` because `mypy` tried to check the file. I think `pytest` doesn't re-test files that haven't changed, and didn't consider removal of `pytest.ini` to invalidate the previous results?

Reply:
Okay, thank you. I didn't know about it.